src/index.html

```diff
@@ -11,7 +11,195 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="#"
+      method="post"
+    >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+
+        <div class="form-field">
+          Surname:
+
+          <label>
+            <input
+              type="text"
+              name="surName"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+
+        <div class="form-field">
+        Name:
```

Comment:
Please move label text inside label tag, this is necessary so that when you click on this text, the input becomes in focus. Please fix for all fields!
![image](https://user-images.githubusercontent.com/54936044/232626208-cc020f06-97b1-46a0-a81e-4849c2d6b55e.png)